spec/support/features/idv_helper.rb

```diff
@@ -39,6 +39,10 @@ def click_idv_continue
     click_spinner_button_and_wait t('forms.buttons.continue')
   end
 
+  def click_idv_update
+    click_spinner_button_and_wait t('forms.buttons.submit.update')
```

Comment:
Is the Ssn page Submit/Update button a spinner button?

Reply:
@soniaconnolly  No.  Updated with commit 3372c4d9f2b03c8126e0d10d57ac000f477ce75c to be...

`  def click_idv_update
    click_on t('forms.buttons.submit.update')
  end
 `
  Thanks for the review!